rpcclient/get_offers.go

```diff
@@ -1,21 +1,20 @@
 package rpcclient
 
 import (
-	"github.com/athanorlabs/atomic-swap/common/types"
 	"github.com/athanorlabs/atomic-swap/rpc"
 )
 
 // GetOffers calls swap_getOffers.
-func (c *Client) GetOffers() ([]*types.Offer, error) {
+func (c *Client) GetOffers() (*rpc.GetOffersResponse, error) {
```

Comment:
We are returning the full response type now, because it now includes the local Peer ID. The Peer ID is not included in the request, so there is a good chance that the caller does not have it.